src/spring/azext_spring/commands.py

```diff
@@ -332,6 +340,31 @@ def load_command_table(self, _):
         g.custom_command('unbind', 'api_portal_custom_domain_unbind')
         g.custom_command('update', 'api_portal_custom_domain_update')
 
+    with self.command_group('spring application-accelerator',
+                            custom_command_type=application_accelerator_cmd_group,
+                            is_preview=True,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_show_command('show', 'application_accelerator_show', table_transformer=transform_application_accelerator_output)
+        g.custom_command('create', 'application_accelerator_create', table_transformer=transform_application_accelerator_output, supports_no_wait=True)
+        g.custom_command('delete', 'application_accelerator_delete', supports_no_wait=True)
+
+    with self.command_group('spring application-accelerator predefined-accelerator',
+                            custom_command_type=application_accelerator_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('list', 'predefined_accelerator_list', table_transformer=transform_predefined_accelerator_output)
+        g.custom_show_command('show', 'predefined_accelerator_show', table_transformer=transform_predefined_accelerator_output)
+        g.custom_command('disable', 'predefined_accelerator_disable', supports_no_wait=True)
+        g.custom_command('enable', 'predefined_accelerator_enable', supports_no_wait=True)
+
+    with self.command_group('spring application-accelerator customized-accelerator',
+                            custom_command_type=application_accelerator_cmd_group,
+                            exception_handler=handle_asc_exception) as g:
+        g.custom_command('list', 'customized_accelerator_list', table_transformer=transform_customized_accelerator_output)
+        g.custom_show_command('show', 'customized_accelerator_show', table_transformer=transform_customized_accelerator_output)
+        g.custom_command('create', 'customized_accelerator_upsert', supports_no_wait=True, validator=validate_customized_accelerator)
+        g.custom_command('update', 'customized_accelerator_upsert', supports_no_wait=True, validator=validate_customized_accelerator)
+        g.custom_command('delete', 'customized_accelerator_delete', supports_no_wait=True)
```

Comment:
Do we need to consider adding `confirmation=true` for those `delete` command?

Reply:
Updated. Add it to application accelerator delete command, as it will delete all of its sub resources.